assets/scss/styles.scss

```diff
@@ -15,52 +16,74 @@ $padding: 15px;
 }
 .column-half {
   @include grid-column( 1/1 );
-  padding: $padding;
+  position: relative;
 }
 .column-two-thirds {
   @include grid-column( 2/3 );
-  padding: $padding;
+  position: relative;
+}
+.column-three-quarters {
+  @include grid-column( 3/4 );
+  position: relative;
 }
 
 /* Header overrides */
+.phase-banner  {
+  @include phase-banner($state: beta);
+}
 @media (min-width: 480px) {
-  #global-header.with-proposition .header-wrapper .header-global {
-    float: left;
-    width: 10%;
-  }
-  #global-header.with-proposition .header-wrapper .header-proposition {
-    width: 90%;
-    float: left;
-  }
-  #global-header .header-proposition {
-    padding-top: 0;
-  }
-  #global-header .header-proposition #proposition-name {
-    font-family: "nta", Arial, sans-serif;
-    font-size: 36px;
-    line-height: 1.25;
-    font-weight: 400;
-    text-transform: none;
-    font-weight: bold;
-    color: white;
-    text-decoration: none;
+  #global-header {
+    &.with-proposition {
+      .header-wrapper {
+        .header-global {
+          float: left;
+          width: 10%;
+        }
+        .header-proposition {
+          width: 90%;
+          float: left;
+        }
+      }
+    }
+    .header-proposition {
+      padding-top: 0;
+      #proposition-name {
+        @include bold-36;
+      }
+      #proposition-link,
+      #proposition-links {
+        a {
+          @include bold-19;
+        }
+      }
+      a.menu {
+        @include core-19;
+        &:after {
+          display: inline-block;
+          font-size: 10px;
+          height: 10px;
+        }
+      }
+    }
   }
 }
 
-h1 {
-  padding: 0 $padding;
-  font-size: 36px;
+@mixin updated_timestamp {
+  text-align: right;
+  color: $red;
+  position: absolute;
+  top: 1.25em;
+  right: 1.25em;
 }
 
 /* Dashboard */
-#centres {
+.centre_data {
   padding: 0;
   h2 {
-    padding-bottom: $padding+5;
-    line-height: 24px;
+    padding-bottom: $padding;
   }
   .panel {
-    background-color: #DEE0E2;
+    background-color: $background-color;
     margin: 0 50px 35px;
```

Comment:
I'm using the $padding variable (set to 1em), and performing maths on it as required for desired padding. No more px.